admin_ui_support/src/Plugin/rest/resource/SimpleConfigResource.php

```diff
@@ -139,6 +151,12 @@ public function availableMethods() {
    * {@inheritdoc}
    */
   public function calculateDependencies() {
+    // @todo Is there a better way to determine the module that provides the
+    // config?
+    $module_name = explode('.', $this->getPluginDefinition()['config_name'])[0];
+    if ($this->moduleHandler->moduleExists($module_name)) {
```

Comment:
Don't we want to add the dependencies all the time, no matter whether the module is installed or not?